handlers/svg-open-street-map/osm-svg.py

```diff
@@ -0,0 +1,216 @@
+        scaled_latitude = dimensions[1] / (lat_max - lat_min)
+        bounds = [[lon_min, lat_min],
+                  [lon_min, lat_max],
+                  [lon_max, lat_max],
+                  [lon_max, lat_min],
+                  [lon_min, lat_min]]
+        # Draw bounding box for the streets
+        for index in range(len(bounds) - 1):
+            p = draw.Path(stroke="orange", stroke_width=6, fill='none')
+            p.M(scaled_longitude * (bounds[index][0] - lon_min),
+                scaled_latitude * (bounds[index][1] - lat_min))
+            p.L(scaled_longitude * (bounds[index + 1][0] - lon_min),
+                scaled_latitude * (bounds[index + 1][1] - lat_min))
+            svg.append(p)
+
+        colors = [
+            "red",
+            "blue",
+            "springgreen",
+            "deeppink",
+            "orange",
+            "purple",
+            "cyan",
+            "coral",
+            "teal",
+            "indigo",
+            "lime",
+            "chocolate",
+            "magenta",
+            "crimson",
+            "deepskyblue",
+            "greenyellow",
+            "gold",
+            "green",
+            "aqua",
+            "navy",
+            "royalblue",
+            "forestgreen",
+            "dodgerblue"
+        ]
+        # Draw the streets with svg.
+        for street in range(len(streets)):
+            color = street
+            if street > len(colors) - 1:
+                color = len(colors) - 1
+            p = draw.Path(
+                stroke=colors[color],
+                stroke_width=1.5,
+                fill='none')
+            node_coordinates = [[node["lon"], node["lat"]]
+                                for node in streets[street]["nodes"]]
+            for index in range(len(node_coordinates) - 1):
+                p.M(scaled_longitude *
+                    (node_coordinates[index][0] -
+                     lon_min), scaled_latitude *
+                    (node_coordinates[index][1] -
+                     lat_min))
+                p.L(scaled_longitude *
+                    (node_coordinates[index +
+                                      1][0] -
+                     lon_min), scaled_latitude *
+                    (node_coordinates[index +
+                                      1][1] -
+                     lat_min))
+                svg.append(p)
+            if "street_name" in streets[street]:
+                svg_layers.append(
+                    {"label": streets[street]["street_name"],
+                        "svg": svg.asDataUri()})
+            else:
+                svg_layers.append(
+                    {"label": str(streets[street]["street_id"]),
+                        "svg": svg.asDataUri()})
+        data = {
+            "layers": svg_layers
+        }
+        rendering = {
+            "type_id": "ca.mcgill.a11y.image.renderer.SVGLayers",
+            "description": "This is SVG data to visualize response \
```

Comment:
Be more specific. This is only showing ways and a label for them - other data in OSM is not shown!